pkg/lint/lintersdb/validator.go

```diff
@@ -75,3 +78,35 @@ func (v Validator) validatePresets(cfg *config.Linters) error {
 
 	return nil
 }
+
+func (v Validator) alternativeNamesDeprecation(cfg *config.Linters) error {
+	if v.m.cfg.InternalTest || v.m.cfg.InternalCmdTest || os.Getenv(logutils.EnvTestRun) == "1" {
+		return nil
+	}
+
+	altNames := map[string][]string{}
+	for _, lc := range v.m.GetAllSupportedLinterConfigs() {
+		for _, alt := range lc.AlternativeNames {
+			altNames[alt] = append(altNames[alt], lc.Name())
+		}
+	}
+
+	var names []string
+	names = append(names, cfg.Enable...)
```

Comment:
For consistency with https://github.com/golangci/golangci-lint/blob/master/pkg/lint/lintersdb/validator.go#L43-L44:
```suggestion
	names := append([]string{}, cfg.Enable...)
```

In the future, it will be replaced with `slices.Concat`.

Reply:
I'm unsure why you simply didn't use

```go
names := cfg.Enable
```

Reply:
Because it will modify `cfg.Enable`